File: sic-api/Entities/Su/SuBusinessAudit.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace sic_api.Entities.Su;

[Table("su_business_audit")]
public class SuBusinessAudit : BaseEntity
{
    [Required]
    [MaxLength(100)]
    [Column("user_id")]
    public string UserId { get; set; } = default!;

    [MaxLength(100)]
    [Column("session_id")]
    public string SessionId { get; set; } = default!;

    [MaxLength(100)]
    [Column("username")]
    public string? Username { get; set; }

    [Required]
    [ForeignKey(nameof(Business))]
    [Column("business_id")]
    public Guid BusinessId { get; set; }
    public SuBusiness Business { get; set; } = default!;

    [MaxLength(50)]
    [Column("client_ip")]
    public string? ClientIp { get; set; }

    [Required]
    [Column("is_active")]
    public bool IsActive { get; set; } = false;

    [MaxLength(500)]
    [Column("remark")]
    public string? Remark { get; set; }
}

File: sic-spring/sic/src/main/java/com/softinter/sicapi/entity/su/SuBusinessAudit.java
package com.softinter.sicapi.entity.su;

import com.softinter.sicapi.entity.base.BaseEntity;
import jakarta.persistence.*;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "su_business_audit")
@Data
@EqualsAndHashCode(callSuper = true)
@NoArgsConstructor
public class SuBusinessAudit extends BaseEntity {

    @Column(name = "user_id", nullable = false, length = 100)
    private String userId;

    @Column(name = "session_id", length = 100)
    private String sessionId;

    @Column(name = "username", length = 100)
    private String username;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "business_id", nullable = false)
    private SuBusiness business;

    @Column(name = "client_ip", length = 50)
    private String clientIp;

    @Column(name = "is_active", nullable = false)
    private Boolean isActive = false;

    @Column(name = "remark", length = 500)
    private String remark;
}